test/e2e/data/infrastructure-aws/kustomize_sources/topology/clusterclass-quick-start.yaml

```diff
@@ -166,6 +203,7 @@ spec:
       # instanceType is a required field (OpenAPI schema).
       instanceType: REPLACEME
       iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
+      cloudInit: {}
```

Comment:
What will happen if we don't pass this? As this is anyways empty so can't we just ignore it?

Reply:
Ohky, thanks for enlightening